exercises/concept/need-for-speed/.meta/design.md

```diff
@@ -17,6 +17,7 @@
 - Inheritance.
 - Finalizers.
 - Method overloading.
+- Loops.
```

Comment:
I don't think we need to list this here. The out-of-scope section is typically used to clarify which parts of the concept being taught are considered out of scope for the exercise. Loops are not really part of the constructors concept, so no need to mention it.

Reply:
👍